ML/PYTORCH/Source/main.cpp

```diff
@@ -164,23 +164,25 @@ void main_main ()
 
     Print() << "Model loaded.\n";
 
+    BL_PROFILE_VAR_STOP(LoadPytorch);
+    
 #ifdef AMREX_USE_CUDA
+    BL_PROFILE_VAR("LoadPytorch",LoadPytorchGpu);
```

Comment:
I think this should be named "LoadPytorchGpu", and on line 174, it should also use LoadPytorchGpu

Reply:
oops... thanks for catching that!